webprojects/SUMO.md

```diff
@@ -0,0 +1,77 @@
+# Support.mozilla.org(SUMO): how to localize and test
+
+SUMO, short for **SU**pport.**MO**zilla.org, is a knowledge base site, providing support to Mozilla product users. More than half of all Firefox users speak a language other than English. Mozilla depends on contributors to make support available to people around the globe.
+
+SUMO consists of two parts:
+* User interface (or UI), such as navigation and buttons.
+* Support articles, either use to troubleshoot issue or as knowledge base for Mozilla products.
+
+This document only covers the UI portion localizable in Pontoon.
+
+## Localizing the strings
+
+There are two active components in Pontoon that make up the [SUMO product](https://pontoon.mozilla.org/sumo):
+
+* **LC_MESSAGES/djangojs.po** contains strings for the JS bits of the platform.
+* **LC_MESSAGES/django.po** contains the majority of strings in the SUMO product.
+
+SUMO is made up by several file types. In order to minimize revision and get translation right the first time, the key is to get the context right. Look for clues or ask others for help:
+* [String ID](https://github.com/mozilla-l10n/sumo-l10n/blob/master/en_US/LC_MESSAGES/django.po#L265) shows where the strings is used.
+* [Comment to a string](https://github.com/mozilla-l10n/sumo-l10n/blob/master/en_US/LC_MESSAGES/django.po#L9274).
```

Comment:
Comments to a string -> String comments

Explain where they're displayed in Pontoon's UI, and why they can be useful. Again, don't point to GitHub or Pontoon, since this link is going to become obsolete in weeks.

Truth is that localization comments in SUMO are non existent. So, the only information useful is:

```
* String comments: in Pontoon they’re displayed above the area where you can enter your translation for a string. They always include a reference to the code where the string is used, and you can use this information to search for context in the [Kitsune GitHub repository](https://github.com/mozilla/kitsune).
```
```